src/commands/createHttpFunction.ts

```diff
@@ -22,60 +21,18 @@ export async function createHttpFunction(context: IActionContext): Promise<void>
 
     const funcApi: AzureFunctionsExtensionApi = await getFunctionsApi(context);
 
-    const endpointName: string = 'endpoint';
```

Comment:
If you want to add this as an option to the api, I'm fine with that

Reply:
You mean like a default name that gets run through `getUniqueFunctionName`

Reply:
Yeah - just something that replaces "HttpTrigger"